react-hot-button-component/src/hot-button.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+export default class HotButton extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.count = 0;
+    this.state = {
+      temp: 'ice'
+    };
```

Comment:
This is okay but for future consideration, it would be better to keep the `count` in state and not the `temp`. The click count is what directly controls what is being rendered. Whatever is controlling what is being rendered is what should be in state.